docs/source/distributed.checkpoint.rst

```diff
@@ -1,4 +1,63 @@
-Distributed Checkpoint
-========================
+Distributed Checkpoint - torch.distributed.checkpoint
+=====================================================
 
-.. automodule:: torch.distributed.checkpoint
```

Comment:
I think we need this line or '.. py:module:: mod_name'.

This is from the test: https://github.com/pytorch/pytorch/actions/runs/3989593761/jobs/6842526619

```
You added the following module(s) to the PyTorch namespace 'torch.distributed.checkpoint' but they have no corresponding entry in a doc .rst file. You should either make sure that the .rst file that contains the module's documentation properly contains either '.. automodule:: mod_name' (if you do not want the paragraph added by the automodule, you can simply use '.. py:module:: mod_name')  or make the module private (by appending an '_' at the beginning of its name).++ exit 1
```

Reply:
trying to figure this out.